### ISSUE TITLE
Can you let me know what excjact means of 0 and -1 in entity.

### ISSUE THREAD

**Issue by JigarPrajpati** [dj-nitehawk/MongoDB.Entities#169]
` .Modify(x => x.Items[-1].isDeleted, true)`

`.Modify(x => x.Items[0].isDeleted, true)`

what actully differnet between these two 0 and -1

**Comment by dj-nitehawk** [comment]
explained here:
https://github.com/dj-nitehawk/MongoDB.Entities/issues/157#issuecomment-1129548681

**Comment by JigarPrajpati** [comment]
Yes sir i knew it thanks you but i've such following trouble, where i want to get only isDeleted==false record, but i can't get any entry.

  `  return Ok(await DB.Find<tblOrder>().Match(x => x.ID == orderInternalID && x.Items[0].isDeleted==false).ExecuteFirstAsync());
        `

1 record
![image](https://user-images.githubusercontent.com/91531559/175513472-84004209-3a07-42b6-8231-b03081105b46.png)

2 record
![image](https://user-images.githubusercontent.com/91531559/175513513-bfd52e9c-dfe3-4524-81cf-13ac49bc19ef.png)
